### ISSUE TITLE
Make a better name

### ISSUE THREAD

**Issue by Anatoliy6463** [SpringTech-Studios/SisijaFight#5]
Sisi on Russian language is a swear word that means "tits". For the same reasons, a Linux distro called ebuntu(ebat on Russian means f**k) was renamed to elbuntu

**Comment by Anatoliy6463** [comment]
And people from Russia love games, so, let's rename this game

**Comment by Lulu-Games** [comment]
We could give a special name in Russian

**Comment by Anatoliy6463** [comment]
> We could give a special name in Russian 

Okay, so what Sisija means?

**Comment by Lulu-Games** [comment]
The Polar bear (main character) is named Sisija

**Comment by Anatoliy6463** [comment]
Nice, look at the difference between STK 0.3 and 1.0, they are the different GAMES

**Comment by Anatoliy6463** [comment]
> Sisi on Russian language is a swear word that means "tits". For the same reasons, a Linux distro called ebuntu(ebat on Russian means f**k) was renamed to elbuntu

Imagine using a distro called f**kbuntu

**Comment by Lulu-Games** [comment]
I wouldnt use that

**Comment by Lulu-Games** [comment]
> Nice, look at the difference between STK 0.3 and 1.0, they are the different GAMES

Yes that actually is right but everyone can name versions like he wants
